x/blockchain/blockchain.go

```diff
@@ -0,0 +1,31 @@
+package blockchain
```

Comment:
Is the objective of this module to wrap the finality storage?

Reply:
Could be, I just needed temporary blockchain functions